Rakefile

```diff
@@ -6,6 +6,21 @@ def usage
     Usage:
       bundle exec rake
 
+      There are three environment variables that control how the tests are run.
+
+      TEST_REPO: This is the repository which will be tested.
+      CORE_REPO: If TEST_REPO is a plugin and you need to specify a different core branch
+      GEM_REPOS: Optional a comma-separated-list of other plugins/gems which are needed to run the tests
```

Comment:
`Optional, a comma-separated list of ...`

Reply:
:+1: done